Mathlib/Tactic/Polyrith.lean

```diff
@@ -390,9 +390,8 @@ Notes:
 * This tactic only works with a working internet connection, since it calls Sage
   using the SageCell web API at <https://sagecell.sagemath.org/>.
   Many thanks to the Sage team and organization for allowing this use.
-* This tactic assumes that the user has `python3` installed and available on the path.
-  (Test by opening a terminal and executing `python3 --version`.)
-  It also assumes that the `requests` library is installed: `python3 -m pip install requests`.
+* This tactic assumes that the user has `python3` installed with version at least 3.6,
```

Comment:
Actually, `request` apparently required Python 3.8 --- so I would definitely increase the requirement here.
Given that Python 3.8 is end-of-live, I'd be happy increasing the requirement to 3.9.

That said: you're not checking that version anywhere, are you?

Reply:
I am not checking that version anywhere.

I don't think request requires Python 3.8. `urllib.request` was added with Python 3.

Reply:
If you mean the package `requests` that was removed, it has [versions](https://pypi.org/project/requests/2.17.3/) for Python all the way back to Python 2. Although probably those versions are not actively maintained anymore.

Reply:
In other words: you could also say "has Python 3 installed" --- which is deliberately unclear, but under most readings does not include end-of-life'd versions. Perhaps that's better; what do you think?

Reply:
Ah, definitely that makes sense!

Reply:
The documentation is now reverted to `python3` installed and available on path